workshops/boost-research-reproducibility-binder/paper.md

```diff
@@ -0,0 +1,116 @@
+
+The feedback report summarises our experience of using these materials in teaching and learning situations. On average, participants report an increase  in their understanding and confidence in using Binder, capturing their computational environment and reproducing their research. Areas that participants identified could be improved include highlighting the difficulties using R with Binder and in particular the tidyverse package which takes a long time to build, that the demonstrations in the afternoon were slower and that some time was taken during the workshop to explain Git terminology,  Markdown and help people get set up.
+
+# Statement of Need
+
+Reproducible research is necessary to ensure that scientific work can be trusted. 
+Funders and publishers are beginning to require that publications include access to the underlying data and the analysis code. 
+The goal is to ensure that all results can be independently verified and built upon in future work, which is sometimes easier said than done. 
+Sharing these research outputs means understanding data management, library sciences, software development, and continuous integration techniques: skills that are not widely taught or expected of academic researchers and data scientists. 
+
+*The Turing Way: A Handbook for Reproducible Data Science* [@the_turing_way_community_2019_3233986] is an open source handbook to support students, their supervisors, funders and journal editors in ensuring that reproducible data science is "too easy not to do". 
+The goal is to provide all the information that researchers need at the start of their projects to ensure that they are easy to reproduce at the end. 
+The [handbook](https://the-turing-way.netlify.com) currently includes chapters and training material on reproducibility, open research, version control, collaborating on GitHub/GitLab, research data management, reproducible environments, analysis testing, reviewing, continuous integration, reproducible research with Make and risk assessment. 
+Further chapters in development include coding style for reproducibility, credit for reproducible research and reproducible data analysis pipelines for machine learning, and will build upon Alan Turing Institute case studies. 
+The project demonstrates open and transparent project management and communication with future users, as it is openly developed at the [GitHub repository](https://github.com/alan-turing-institute/the-turing-way). 
+It also facilitates workshops to train researchers and research software engineers to use tools to make their work more reproducible - one of which is the *Boost your research reproducibility with Binder* workshop.
+
+The Binder project significantly lowers the bar to sharing and re-using software, as it allows a user to try out someone else's work simply by clicking a single link. 
```

Comment:
Am just wondering whether sharing and reusing is the leading benefit of binder? It's my impression that github, good documentation and dependency management are the main drivers of sharing and reusability. To me, binder supports the ability to instantly reproduce and interact with shared code. Admittedly this might just be a comment on semantics. I generally use the terms as described here: https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5758530/

Reply:
Thanks @annakrystalli, I've added that in!